events/kwoc.md

```diff
@@ -0,0 +1,99 @@
+## First things first 
+
+- KWoC is a month long event, generally conducted in December.
+- Registrations start atleast a week before the coding period(More than a week is preferred)
+- Coding period is generally 4 weeks long
+- Mid evals are held after around 2 weeks of coding period
+- End evals are done after the coding period
+
+## Pre-registration
+
+- The following things should be ready and updated:-
+
+  - Website with registration feature tested internally.
+  - FAQs on the website
+  - Student and Mentor Docs of KWoC. Make sure that the content of the doc is updated with the current rules of KWoC(especially rules like Mid Evals and End Evals criteria). If there are any new important changes, do mention them in the docs. 
+
+- A dedicated slack workspace for KWoC. If the old one has reached the maximum free limit, create a new Slack workspace.
```

Comment:
@Shankusu993 was suggesting about having a discord server. Advantages that he mentioned:
- No limit on number of users
- Easy to manage roles


But I say Slack is better because, for every query there can be a thread created. But discord doesn't have threads. After 10 or 20 messages, the query might be lost. And if a query is unanswered, the defeats the purpose of a platform for queries(other than mail/Github). Others, please give your opinion

Reply:
Another important reason for me suggesting discord besides the no member limit is to avoid what happened a year ago where a person kept spamming the channel and created disturbance for everyone. When banned he created new accounts and proceeded to do the same. In discord it's easy to atleast  avoid spam part by having slow mode on a channel.

Reply:
> It doesn't make much of a difference

No, it makes a difference. Discord doesn't provide the feature of threads. If after a query there are some 50 messages on some other topic it gets lost. With threads that does't happen.

Also, I and @sahil-shubham checked. There is no limit on the number of users in slack. 

Spamming happens very rarely. It happened once in 2 years. That too on one part of the day. Clarifying doubts with ease is the main aim of a common group. We can't sacrifice it.

Reply:
> No, it makes a difference.

Oh, yeah! It matters, my bad. I meant in a sense that it doesn't matter much to me or in the grand scheme of KWoC (except threads as you pointed out). So, I didn't care much.